pom.xml

```diff
@@ -128,6 +129,11 @@
             <version>${testcontainers.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.modelmapper</groupId>
+            <artifactId>modelmapper</artifactId>
+            <version>${modelmapper.version}</version>
+        </dependency>
```

Comment:
This dependency should be moved up to jackson and lombok for dependency organisation